web/rootfs/defaults/settings-config.js

```diff
@@ -386,9 +388,24 @@ config.hiddenPremeetingButtons = [ '{{ join "','" (splitList "," .Env.HIDE_PREME
 
 // Configure remote participant video menu
 if (!config.hasOwnProperty('remoteVideoMenu')) config.remoteVideoMenu = {};
+config.remoteVideoMenu.disabled = {{ $DISABLE_REMOTE_VIDEO_MENU }};
 {{ if .Env.DISABLE_KICKOUT -}}
```

Comment:
Can you please apply the same pattern to these other 2 related options?

Reply:
Please check if the defaults are fine
DISABLE_KICKOUT=false
DISABLE_GRANT_MODERATOR=false